aruco_markers:
  - id: 0
    parent_link: "R5A_link1"
    translation: [-0.120, -0.030, 0.0]
    rotation: [1.5708, 1.5708, 0]  # Roll, Pitch, Yaw in radians
  - id: 1
    parent_link: "R5A_link1"
    translation: [0.120, -0.030, -0.0]
    rotation: [-1.5708, -1.5708, 0]
  - id: 2
    parent_link: "R5A_link1"
    translation: [-0.140, 0, 0]
    rotation: [0, 1.5708, 0]
  - id: 3
    parent_link: "R5A_link1"
    translation: [0.140, 0, 0]
    rotation: [0, -1.5708, 0]

  - id: 4
    parent_link: "R5A_link2"
    translation: [0, 0, 0.040]
    rotation: [0, 3.14, 0]
  - id: 5
    parent_link: "R5A_link2"
    translation: [0, 0, -0.040]
    rotation: [0, 0, 0]
  - id: 6
    parent_link: "R5A_link2"
    translation: [-0.06, -0.05, 0]
    rotation: [0, 1.5708, 0]
  - id: 7
    parent_link: "R5A_link2"
    translation: [0.08, 0, 0]
    rotation: [0, -1.5708, 0]

  - id: 8
    parent_link: "R5A_link3"
    translation: [-0.04, 0, 0]
    rotation: [0, 1.5708, 0]
  - id: 9
    parent_link: "R5A_link3"
    translation: [0, 0.065, 0.035]
    rotation: [0, 3.14, 0]
  - id: 10
    parent_link: "R5A_link3"
    translation: [0, 0.065, -0.035]
    rotation: [0, 0, 0]
  - id: 11
    parent_link: "R5A_link3"
    translation: [0.04, 0, 0]
    rotation: [0, -1.5708, 0]

  - id: 12
    parent_link: "R5A_link4"
    translation: [0, 0, 0.035]
    rotation: [0, 3.14, 0]
  - id: 13
    parent_link: "R5A_link4"
    translation: [0.035, 0, 0]
    rotation: [0, -1.5708, 0]
  - id: 14
    parent_link: "R5A_link4"
    translation: [-0.035, -0.075, 0]
    rotation: [0, 1.5708, 0]
  - id: 15
    parent_link: "R5A_link4"
    translation: [0, 0.030, -0.035]
    rotation: [0, 0, 0]
